### PR TITLE
Client side captcha

### PR DESCRIPTION
**Pull request by amisha11**
This time I have added google recaptcha which is purely client side.
You will have to generate a site-key (link given as a comment) and the captcha will be able to identify bots.

### CODE REVIEW

File: captcha_code_file.php [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<?php
```
Comment (richardwestenra) [inline captcha_code_file.php]: Can you please delete this PHP file?

File: index.html [hunk 2]
```diff
@@ -265,6 +278,11 @@ <h2 class="bannerbox__title">I need a ride</h2>
                             </label>
                             <small>Yes, please stay in touch!</small>
                         </div>
+						<div class="form-group">
+							<script src='https://www.google.com/recaptcha/api.js'></script>
+							<div class="g-recaptcha" data-sitekey="YOUR_KEY_1"></div> <!--Generate your site owner key (http://www.google.com/recaptcha/intro/) and add here-->
```
Comment (richardwestenra) [inline index.html]: We're going to need to generate a site key for you. I'll get back to you about this.

File: index.html [hunk 3]
```diff
@@ -26,6 +26,19 @@
...
       <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
+	
+	<script>
+		function get_action(form) {
+
+		var v = grecaptcha.getResponse();
+		if(v.length == 0)
+		{
+			document.getElementById('captcha').innerHTML="You left the Captcha Code empty";
```
Comment (richardwestenra) [inline index.html]: Using an ID here might not work, because there's more than one on the page. Needs testing though, once we've got the key added.

Comment (amisha11) [inline index.html]: I have deleted captcha_code_file.php
For now I have commented out the validation js I'll try to look for some other way of doing that

**Pull request closed without merging**